msresist/figures/figureM3.py

```diff
@@ -40,13 +40,12 @@ def makeFigure():
     tt = centers.iloc[:, -1]
     tt = tt.replace("Normal", 0)
     tt = tt.replace("Tumor", 1)
-    lr = LogisticRegressionCV(cv=model.ncl, solver="saga", penalty="l1").fit(c, tt)
-    y_pred = lr.predict(c)
+    lr = LogisticRegressionCV(cv=model.ncl, solver="saga", penalty="l2").fit(c, tt)
```

Comment:
LASSO was setting all coefficients to 0 except from cluster 13. Using L2, we still see the same cluster 13 effect, but also identifies further clusters as being relevant.

Reply:
I actually think that's a good thing, because then you just have to look at cluster 13 to explain the prediction. However, I adjusted the settings a bit so that it tries two weights of l1 vs l2, and takes the one with the least CV error.